src/main/java/com/ft/wordpressarticlemapper/WordPressArticleMapperApplication.java

```diff
@@ -120,10 +115,9 @@ public void run(final WordPressArticleTransformerConfiguration configuration, fi
                         "https://sites.google.com/a/ft.com/ft-technology-service-transition/home/run-book-library/documentstoreapi",
                         Client.create(), configuration.getUrlResolverConfiguration().getDocumentStoreConfiguration().getEndpointConfiguration()));
 
-        environment.jersey().register(WordPressArticleTransformerExceptionMapper.class);
         Errors.customise(new WordPressArticleMapperErrorEntityFactory());
         environment.servlets().addFilter("Transaction ID Filter",
-                new TransactionIdFilter()).addMappingForUrlPatterns(EnumSet.allOf(DispatcherType.class), false, "/content/*");
+                new TransactionIdFilter()).addMappingForUrlPatterns(EnumSet.allOf(DispatcherType.class), false, "/map/*", "/ingest/*");
```

Comment:
The mapping here does not exactly match the annotations in the resource class, which don't have wildcards. I think you should use the same values here.